app/app.go

```diff
@@ -883,6 +884,19 @@ func NewApp(
 	app.SetBeginBlocker(app.BeginBlocker)
 	app.SetEndBlocker(app.EndBlocker)
 
+	app.upgradeKeeper.SetUpgradeHandler(
+		upgradeName,
+		func(ctx sdk.Context, _ upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+			fromVM := app.mm.GetVersionMap()
+			fromVM[incentivetypes.ModuleName] = 1
+			fromVM[auctiontypes.ModuleName] = 1
+			delete(fromVM, authz.ModuleName)
+			delete(fromVM, bridgetypes.ModuleName)
+			delete(fromVM, savingstypes.ModuleName)
```

Comment:
```suggestion
		func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
```

Not necessary for testnet, but I think you shouldn't need to manually set these if you read `fromVM` from the func arg and you remove line 30 (`app.upgradeKeeper.SetModuleVersionMap(ctx, app.mm.GetVersionMap())`).